packages/libs/react-ui/src/components/Select/Select.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { Select } from '@components/Select';
+import { Option } from '@components/Select/Option';
+import { fireEvent, render } from '@testing-library/react';
+import React from 'react';
+
+describe('Select', () => {
+  it('renders without errors', () => {
+    const { getByTestId } = render(
+      <Select value="1" onChange={() => {}}>
+        <Option value="1">Option 1</Option>
+        <Option value="2">Option 2</Option>
+      </Select>,
+    );
+
+    const selectContainer = getByTestId('kda-select');
+    expect(selectContainer).toBeInTheDocument();
+  });
+
+  it('renders the provided children options', () => {
+    const { getByTestId } = render(
+      <Select value="1" onChange={() => {}}>
+        <Option value="1">Option 1</Option>
+        <Option value="2">Option 2</Option>
+      </Select>,
+    );
+
+    const selectContainer = getByTestId('kda-select');
+    const selectElement = selectContainer.querySelector('select');
+    const option1 = selectContainer.querySelector('option[value="1"]');
+    const option2 = selectContainer.querySelector('option[value="2"]');
+
+    expect(selectElement).toBeInTheDocument();
+    expect(option1).toBeInTheDocument();
+    expect(option2).toBeInTheDocument();
```

Comment:
it's better to avoid normal DOM query selectors in test as much as possible. this could be written like below
```
expect(screen.getByRole('combobox')).toBeInTheDocument();
    expect(
      screen.getByRole('option', { name: 'Option 1' }),
    ).toBeInTheDocument();
    expect(
      screen.getByRole('option', { name: 'Option 2' }),
    ).toBeInTheDocument();
    expect(
      (screen.getByRole('option', { name: 'Option 1' }) as HTMLOptionElement)
        .selected,
    ).toBe(true);
```

the tests below as well could be re-written in same format.

Reply:
I have tried but it doesn't work
![image](https://github.com/kadena-community/kadena.js/assets/29011692/3f34b592-dd7e-40e2-8973-76dd851f640e)

also, it's a common practice to use query selector in jest, + this won't improve coverage or benefit in any way